internal/gitinterface/commit.go

```diff
@@ -210,6 +211,37 @@ func (r *Repository) CommitUsingSpecificKey(treeID Hash, targetRef, message stri
 	return commitIDHash, r.CheckAndSetReference(targetRef, commitIDHash, refTip)
 }
 
+// CommitWithParents creates a new commit in the repo but does not update any
+// references. It is only meant to be used for tests, and therefore accepts
+// specific parent commit IDs.
+func (r *Repository) CommitWithParents(t *testing.T, treeID Hash, parentIDs []Hash, message string, sign bool) Hash {
```

Comment:
can we unexport this as well? I think it's only being used in the package.

Reply:
I originally had it in _test but then needed it in the rsl package. https://github.com/gittuf/gittuf/pull/372/files#diff-a6428d90ac83e051ab984398f3c2d809cab07eaeb1e7a0378a10806065851b9aR669

Reply:
I could create a commit in a new branch in the rsl package then SetReference() the main branch to that no-parent commit, lmk if that makes sense.